### PR TITLE
Add coffee color scheme from wp-admin

### PR DESCRIPTION
**Pull request by mreishus**
One part of #47036

#### Changes proposed in this Pull Request

* Add coffee color scheme

#### Testing instructions

* Visit /me/account
* Choose coffee and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:

Wp-admin:

#### Notes

If you're following along with Ectoplasm from #47122, you'll see I generated a palette of "Ectoplasm green" in -10 to -90 values.  For this one, I found that "Studio Orange" was a close enough match for primary+accent colors and reused it.  Orange and brown are close to each other in terms of color hue, who would have thunk it?  I'm open to revision or ideas on this, though.

In wp-admin this is a "low contrast" scheme, which means that certain buttons are black instead of the highlight color.  I haven't accounted for that here.

### CODE REVIEW

**Review by sixhours — CHANGES_REQUESTED** [review]
🎉  This is coming together! Just a few minor things. Coffee could benefit from the same border treatment on the top bar as Ectoplasm:

Screen Shot 2020-11-06 at 10 24 21 AM

I'm seeing the dark outline around the inline help button:

Screen Shot 2020-11-06 at 10 06 04 AM

Noticed the gray plan flag in the sidebar could use a color shift:

Screen Shot 2020-11-06 at 10 05 52 AM

And the Reader notification icon is still pink:

Screen Shot 2020-11-06 at 10 14 26 AM

File: packages/calypso-color-schemes/src/shared/color-schemes/_coffee.scss [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    --theme-text-color: #ffffff; /* Direct from wp-admin */
+    --theme-text-color-rgb: 255, 255, 255; /* Manual conversion */
+    --theme-base-color: #59524c; /* Direct from wp-admin */
+    --theme-base-color-rgb: 89, 82, 76; /* Manually computed https://www.rapidtables.com/convert/color/hex-to-rgb.html */
+    --theme-submenu-background-color: #46403c; /* darken( $base-color, 7% ), computed: http://scg.ar-ch.org/
+    --theme-icon-color: #ece6f6; /* Direct from wp-admin */
+    --theme-highlight-color: #c7a589; /* Direct from wp-admin */
+    --theme-highlight-color-rgb: 199, 165, 137; /* Manually computed https://www.rapidtables.com/convert/color/hex-to-rgb.html */
+    --theme-notification-color: #9ea476; /* Direct from wp-admin */
+}
```
Comment (cpapazoglou) [inline packages/calypso-color-schemes/src/shared/color-schemes/_coffee.scss:34]: As also suggested in https://github.com/Automattic/wp-calypso/pull/47122 let's try consolidating these variables in _defaut.scss.

**Review by cpapazoglou — CHANGES_REQUESTED** [review]
1. Sidebar Menu Headings should be white

File: packages/calypso-color-schemes/src/shared/color-schemes/_coffee.scss (hunk 1 above)
Comment (mreishus) [reply packages/calypso-color-schemes/src/shared/color-schemes/_coffee.scss]: Done

**Comment by mreishus** [comment]
For the upsell nudge, I added a custom override.  However, it's limited in scope to that one css class combination.  It doesn't seem to be themeable as calypso hardcodes it to a neutral color:



My override is a 15% darken of the base color:


None of the other themes currently do this.

**Comment by mreishus** [comment]
I believe I've addressed everything except the circle around the FAB Inline Help button (explanation is in the ectoplasm PR #47122)

**Review by cpapazoglou — APPROVED** [review]
Thanks for pushing the changes @mreishus!

LGTM!

**Comment by mreishus** [comment]
The override for the nudge banner is causing issues when the nav unification flag is on:

With Override:


Without Override:


I could make the override to be more specific, but I'm worried about it causing more issues I haven't seen yet, or that it will get lost as we reorganize CSS as the nav unification project continues.  I propose removing the override, sticking with the base neutral colors set, and waiting until we control the banner with theme variables instead of neutral colors. @sixhours @cpapazoglou

**Pull request merged**